ui/prometheus-plugin/package.json

```diff
@@ -34,8 +34,7 @@
     "@perses-dev/components": "0.25.0",
     "@perses-dev/core": "0.25.0",
     "@perses-dev/plugin-system": "0.25.0",
-    "@prometheus-io/codemirror-promql": "^0.40.5",
-    "@prometheus-io/lezer-promql": "^0.37.0",
```

Comment:
Do we not use lezer-promql anymore?

Reply:
not from what I find in the code at least :). We only use codemirror-promql.